app/assets/stylesheets/application.scss

```
@@ -185,6 +177,26 @@ form.search-form input.search-text {
   padding-left: 50px;
 }
 
+form.search-form input::placeholder {
+  font-size:18px
+}
+
+main.default_wrap {
+  margin: 0 auto;
+  padding: 1rem 0;
+  max-width: 910px;
+  display: block;
+  box-sizing: border-box;
+}
+
+main.rails_text_wrap {
+  margin: 0 auto;
+  padding: 1rem 0;
+  max-width: 100%;
+  display: block;
+  box-sizing: border-box;
+}
```

共通部分は`main` にまとめましょう。

```suggestion
main {

}

main.default_wrap {

}

main.rails_text_wrap {

}
```